### PR TITLE
Amendments For UG Add and Delete Sections

### PR DESCRIPTION
**Pull request by HM33-Stu**
Amendments For Add and Delete sections (User Guide) as well as some issues carried over from other PRs

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -316,6 +316,20 @@ you fully understand each command and their usage.
...
+| [delete](#deleting-an-internship-delete)      | Removes a Internship.                    | `delete INDEX` |
+| [list](#listing-all-internships-list)         | Removes a Internship.                    | `list` |
+| [edit](#editing-an-internship-edit)           | Modifies an existing Internship.         | `edit INDEX [/com COMPANY_NAME] [/poc CONTACT_NAME] [/email CONTACT_EMAIL] [/phone CONTACT_NUMBER] [/loc LOCATION_ENUM] [/status STATUS] [/desc DESCRIPTION] [/role ROLE` |
+| [addremark](#adding-a-remark-addremark)       | Adds a remark to an existing Internship. | `addremark INDEX [/remark REMARK]` |
+| [clear](#clearing-all-internships-clear)      | Removes all Internships from the deck.   | `clear` |
+| [find](#finding-internships-by-keywords-find) | Finds an Internship by keywords.         |`find MODE [/com COMPANY_NAME_KEYWORDS] [/poc CONTACT_NAME_KEYWORDS] [/loc LOCATION_KEYWORDS] [/status STATUS_KEYWORDS] [/desc DESCRIPTION_KEYWORDS] [/role ROLE_KEYWORDS]`|
+| [exit](#exiting-the-program-exit)             | Exits and closes the application.        | `exit` |
+
+
```
Comment (HM33-Stu) [inline docs/UserGuide.md:332]: Shifted this above to right after Quick Start (as discussed in the call).
Should we shift Field Summary up here as well?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -344,16 +358,37 @@ Shows a message explaning how to access the help page.
...
 Format: `add /com COMPANY_NAME /desc DESCRIPTION /status STATUS /poc CONTACT_NAME /email CONTACT_EMAIL /phone CONTACT_NUMBER /loc LOCATION_ENUM /role ROLE ​`
 
-Examples:
-* `add /com Tiktok /desc create new recommendation engine /status ongoing /poc jane yeo /email hr@tiktok.com
+* The fields `COMPANY_NAME`, `DESCRIPTION`, `CONTACT_NAME` and `ROLE` allow the use of any text, number and/or spaces
+* The field `STATUS` accepts only the following inputs: `to_apply`, `ongoing`, `rejected`, `accepted`
+* The field `CONTACT_EMAIL` accepts only the format of `<email> @ domain.com`
+* The field `CONTACT_NUMBER` accepts only positive integers and it must be minimally 3 **positive integers** and above
+  * A **positive integer** refers to 1, 2, 3, …​
+* The field `LOCATION_ENUM` accepts only the following inputs: `remote`, `local`, `overseas`
```
Comment (HM33-Stu) [inline docs/UserGuide.md:372]: Valid Parameters For Each Field

File: docs/UserGuide.md [hunk 3]
```diff
@@ -400,7 +442,7 @@ Click the back button to return to the previous view of internship entries.
 
 Shows the full details of an internship entry. This replaces the current view of internship entries.
 
-[CLI](#glossary)input will be implemented in a future release.
+[CLI](#glossary) input will be implemented in a future release.
```
Comment (HM33-Stu) [inline docs/UserGuide.md:445]: Spacing

**Comment by HM33-Stu** [comment]
Please do note that Try-It-Out sections might be taken out as discussed, as we already have them in the tutorial.
Pending decision based on further discussion

**Review by LWS49 — APPROVED** [review]

**Pull request merged**